### PR TITLE
Wip master

### PR DESCRIPTION
**Pull request by jordanhudgens**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -10,10 +8,37 @@
...
+1. User fills out the form on `students/new`
+
+2. When the form is submitted a new record is created in the database
+
+3. After the user fills out the form they are redirected to the `show` page that renders the `student` record that they created
+
+
+## Keys to remember
+
+* Your `create` method should create a new `Student` instance and store it in an instance variable
```
Comment (jmburges) [inline README.md]: Go ahead and drop these keys to remember. Maybe just the line about make sure to use route helpers

File: app/controllers/students_controller.rb [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+  end
+
+  def show
+  end
+
+  def new
+  end
+
+  def create
+    session[:form_params] = params.inspect
```
Comment (jmburges) [inline app/controllers/students_controller.rb]: drop this session stuff in the start code just leave this action empty

File: app/controllers/students_controller.rb [hunk 3]
```diff
@@ -0,0 +1,24 @@
+class StudentsController < ApplicationController
+  before_action :set_student, only: :show
```
Comment (jmburges) [inline app/controllers/students_controller.rb]: also we haven't covered `before_action` so drop that

File: spec/controllers/students_controller_spec.rb [hunk 4]
```diff
@@ -0,0 +1,20 @@
+require 'rails_helper'
+
+describe StudentsController do
+
+  describe "POST create" do
+    context "with valid attributes" do
+      it "creates a new student" do
+        expect{
+          post :create, student: FactoryGirl.attributes_for(:student)
```
Comment (jmburges) [inline spec/controllers/students_controller_spec.rb]: no factory girl

**Comment by jmburges** [comment]
back at you!

**Comment by jordanhudgens** [comment]
@jmburges This has been updated, I also updated the [solution branch](https://github.com/learn-co-curriculum/rails-create-action-lab/tree/wip-solution).

**Pull request merged**